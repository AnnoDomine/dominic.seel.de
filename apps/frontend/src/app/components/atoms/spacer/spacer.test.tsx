import { render, screen } from "@testing-library/react";
import { describe, expect, it } from "vitest";
import Spacer from "./spacer";

describe("Spacer", () => {
    it("renders without crashing", () => {
        render(<Spacer data-testid="spacer" />);
        const spacer = screen.getByTestId("spacer");
        expect(spacer).toMatchSnapshot();
    });
});
