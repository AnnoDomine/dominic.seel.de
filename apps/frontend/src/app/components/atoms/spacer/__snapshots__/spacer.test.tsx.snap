// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Spacer > renders without crashing 1`] = `
<div
  class="css-s3l9e8"
  data-testid="spacer"
/>
`;
